File: scala/scala_vs_c/PiCalculator.java
class PiCalculator {
	public static void main(String []args) {
		long startTime = System.currentTimeMillis();
		int start = 0;
		int nrOfElements = 1000 * 1000 * 100;
		double acc = 0;
		for (int i = start; i < nrOfElements; ++i) {
			acc += (4.0 * (1 - (i % 2) * 2)) / (2 * i + 1);
		}
		System.out.println((System.currentTimeMillis() - startTime) + " ms");
		System.out.println(acc);
	}
}

File: scala/scala_vs_c/PiCalculator.scala
object PiCalculator extends App {
   val startTime = System.currentTimeMillis
   val start = 0
   val nrOfElements = 1000 * 1000 * 100
   var acc: Double = 0
   for (i <- start until (start + nrOfElements))
        acc += 4.0 * (1 - (i % 2) * 2) / (2 * i + 1)
   println((System.currentTimeMillis - startTime) + " ms")
   println(acc)
}
